packages/inter-protocol/src/proposals/replaceElectorate.js

```diff
@@ -220,34 +209,15 @@ const addGovernorsToEconCharter = async (
     E(creatorFacet).addInstance(psm, psmGovernorCreatorFacet, label);
   }
 
-  await Promise.all(
-    [
-      {
-        label: 'reserve',
-        instanceP: reserve,
-        facetP: E.get(reserveKit).governorCreatorFacet,
-      },
-      {
-        label: 'VaultFactory',
-        instanceP: VaultFactory,
-        facetP: E.get(vaultFactoryKit).governorCreatorFacet,
-      },
-      {
-        label: 'auctioneer',
-        instanceP: auctioneer,
-        facetP: E.get(auctioneerKit).governorCreatorFacet,
-      },
-      {
-        label: 'provisionPool',
-        instanceP: provisionPool,
-        facetP: E.get(provisionPoolStartResult).governorCreatorFacet,
-      },
-    ].map(async ({ label, instanceP, facetP }) => {
-      const [instance, govFacet] = await Promise.all([instanceP, facetP]);
+  const governedContractKitMap = await governedContractKits;
 
-      return E(creatorFacet).addInstance(instance, govFacet, label);
-    }),
-  );
+  for (const {
+    instance,
+    governorCreatorFacet,
+    label,
+  } of governedContractKitMap.values()) {
+    E(creatorFacet).addInstance(instance, governorCreatorFacet, label);
```

Comment:
There are lots of contracts floating around. let's rename `creatorFacet` to `ecCreatorFacet` or `committeeCreatorFacet`.

Reply:
Good point. Made the update.